### ISSUE TITLE
Second body color?

### ISSUE THREAD

**Issue by jlconlin** [samcarter/tikzlings#6]
I think it would be great if one could specify a second color tor the body of a tikzling. For example:

```
\bear[body=blue, body2=yellow]
```

I think I could make the change for the bear and put in a Pull Request, but wanted to make sure what the right name for the second color should be. It would also be great if all of the tikzlings could have the same option and I'm not sure the right way to do that.

**Comment by samcarter** [comment]
@jlconlin Thanks for the suggestion! Just a quick question if I understand it correctly: What would be the purpose of the second colour? Would it change the colour of the belly etc.?

**Comment by jlconlin** [comment]
Yeah, change the color of the belly/paws/ears/etc. I should have been more explicit. 

Right now (if I understand correctly) you just take the color given (brown by default) and mix it 50% with white. I think that's a good default, but if I want a different color, that should be doable too, right?

**Comment by samcarter** [comment]
Thanks for the clarification! 

You got me there, this is indeed a point where I have been a bit lazy. For some of the other tikzlings I have individual keywords that control the colour of the belly, muzzle, feet .... but for the bear I skipped these.   

I see two possible approaches:

- (as you suggested) create a key that controls the colour of all highlighted parts at once (for tikzlings also a tertiary colour might be necessary)

- create individual keys for all parts, e.g. belly, muzzle, feet... This will give the most flexible solution, but will also blow up the documentation quite a bit
